src/svelteDirectives/isSSR.ts

```diff
@@ -0,0 +1,7 @@
+export default (function isSSR(): boolean {
```

Comment:
This could be used by anything, doesn't seem svelte-specific, perhaps there's a different directory name we could use.